.gitignore

```diff
@@ -3,3 +3,8 @@
 .tox/
 dist/
 htmlcov/
+
+# Docs
+docs/_build
+docs/_static
+docs/_templates
```

Comment:
_static and _templates are referred to in conf.py; I think they're supposed to be in version control, though we don't have any custom static files at the moment.

Reply:
That's true.

- removed `_static` and `_templates` folders from `.gitignore`
- they won't be committed to the repository because they're empty dirs